cmd/cluster-network-operator/main.go

```diff
@@ -27,30 +31,25 @@ func printVersion() {
 
 const LOCK_NAME = "cluster-network-operator"
 
-// urlOnlyKubeconfig is a slight hack; we need to get the apiserver from the
-// kubeconfig but should use the in-cluster service account
-var urlOnlyKubeconfig string
-
-func init() {
-	flag.StringVar(&urlOnlyKubeconfig, "url-only-kubeconfig", "",
-		"Path to a kubeconfig, but only for the apiserver url.")
-}
+const ENV_URL_KUBECONFIG = "URL_ONLY_KUBECONFIG"
```

Comment:
It can't be a command-line argument due to silliness in how the library-go scaffolding is set up :-/

We're _really_ the only ones who have this problem. Everyone else just talks to either 172.16.0.1 (or so) or 127.0.0.1.

Filed SDN-1347.